src/main/scala/cromwell/binding/values/WdlValueJsonFormatter.scala

```diff
@@ -11,7 +11,11 @@ object WdlValueJsonFormatter extends DefaultJsonProtocol {
       case f: WdlFloat => JsNumber(f.value)
       case b: WdlBoolean => JsBoolean(b.value)
       case f: WdlFile => JsString(f.value)
-      case o: WdlObject => JsObject()
+      case o: WdlObject => new JsObject(
```

Comment:
ooh I see what you did... any reason why it can't serialize to JSON just like a `WdlMap` serializes?